modules/events/application/use-cases/create-occasion.ts

```
@@ -0,0 +1,17 @@
+} from "../ports/occasion-repository";
+
+export interface CreateOccasionDependencies {
+	occasionRepository: OccasionRepository;
+}
+
+export function createCreateOccasion(dependencies: CreateOccasionDependencies) {
+	return async function createOccasion(
+		params: CreateOccasionParams,
+	): Promise<OccasionRecord> {
+		return dependencies.occasionRepository.create(params);
+	};
```

Biome (2.4.4)

[error] This async function lacks an await expression.

(lint/suspicious/useAwait)